### ISSUE TITLE
Urgent notifications can get offset

### ISSUE THREAD

**Issue by jeremypw** [elementary/notifications#169]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Describe the bug
Urgent notifications can become offset after messing with workspaces and displays.

If applicable, add screenshots or screen recordings to help explain your problem.

### To Reproduce
1. Send some Urgent notifications
2. Send some normal notifications 
3. Switch between workspaces and/ or change orientation of display

The exact operations needed to trigger this bug have not been identified - this report will be updated if they are.
### Screenshots or screen recordings

![OffsetNotitifcations](https://user-images.githubusercontent.com/10513844/157075109-c188d7ff-6412-4180-9477-cee3efbf1c9e.png)
## Platform Information

- [x ] I'm using the latest released stable version

**Comment by davidmhewitt** [comment]
I'm pretty sure it's gala that's responsible for positioning notifications.

**Comment by jeremypw** [comment]
Yes, you are right :roll_eyes:  Are you able to transfer this? Gala is not one of the projects I can transfer issues to.
